src/elm/Cambiatus/Mutation.elm

```diff
@@ -5,15 +5,11 @@
 module Cambiatus.Mutation exposing (..)
 
 import Cambiatus.InputObject
-import Cambiatus.Interface
 import Cambiatus.Object
-import Cambiatus.Scalar
-import Cambiatus.ScalarCodecs
-import Cambiatus.Union
 import Graphql.Internal.Builder.Argument as Argument exposing (Argument)
 import Graphql.Internal.Builder.Object as Object
 import Graphql.Internal.Encode as Encode exposing (Value)
-import Graphql.Operation exposing (RootMutation, RootQuery, RootSubscription)
+import Graphql.Operation exposing (RootMutation)
```

Comment:
@marcjal please ignore this if you haven't touched this, but just to let you know in any case:

all files under `src/elm/Cambiatus` are generated by our [GraphQL package](https://github.com/dillonkearns/elm-graphql) that generates strong Elm Types based on our GraphQL schema. Its nice but sometimes the generated code have some unused imports and stuff like this. You can safely ignore them and never change those files

Reply:
I got it! This was my elm-format I think or maybe I delete it when I saw it without use.